dlme_airflow/dags/intake_harvester.py

```diff
@@ -0,0 +1,36 @@
+from airflow import DAG
+from datetime import timedelta
+
+from harvester.csv import csv_harvester
+
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+    "provider": None,
+}
+
+with DAG(
+    "intake-harvester",
+    default_args=default_args,
+    description="Intake Harvester DAG",
+    schedule_interval=timedelta(days=1),
+    start_date=days_ago(2),
```

Comment:
It is recommended to set this to an actual date and add `catchup: False`. The scheduler will always pick a dynamic start date (e.g. this isn't the from when the dag is created it's from when the instance is scheduled) and can lead to unexpected extra task runs.

Reply:
Done, fixed in commit https://github.com/sul-dlss/dlme-airflow/pull/19/commits/9ec8830fc46fa16bf421d258444cb48b98384725